### ISSUE TITLE
modinfo -p zfs/spl to show default values in a parsable form

### ISSUE THREAD

**Issue by mailinglists35** [openzfs/zfs#11686]
### Describe the feature would like to see added to OpenZFS

currently the default values are only found in manual pages and only few are shown in the `modinfo -p` output.

### How will this feature improve OpenZFS?

displaying these values in `modinfo -p` output would make easy to implement a feature such as #11685

**Comment by mailinglists35** [comment]
while looking at modules that have tunable parameters, I found that icp and zcommon do not have manpages for them:

root@zfs:~# modinfo -p zcommon
zfs_max_dataset_nesting:Limit to the amount of nesting a path can have. Defaults to 50. (int)
zfs_fletcher_4_impl:Select fletcher 4 implementation.

root@zfs:~# modinfo -p icp
icp_aes_impl:Select aes implementation.
icp_gcm_impl:Select gcm implementation.
icp_gcm_avx_chunk_size:How many bytes to process while owning the FPU

root@zfs:~# man icp-module-parameters
No manual entry for icp-module-parameters

root@zfs:~# man zcommon-module-parameters
No manual entry for zcommon-module-parameters

**Comment by AttilaFueloep** [comment]
> while looking at modules that have tunable parameters, I found that icp and zcommon do not have manpages for them:

Yes, sorry. Documenting the ICP module parameters is still on my todo list.
